### PR TITLE
Styled the List view

### LINKED ISSUE #51: Style the List view
**Description**
The list view looks great on both Mobile and desktop

**Acceptance Criteria**

- [ ] The List view Looks great in large and small screens, this happens both when there are items on the list and when there arent any items.

### PR DESCRIPTION
**Pull request by vicowiti**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
Styled the List view to reflects the app's style

## Related Issue
closes #51 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

#### Desktop
image
image
#### Mobile
image

image

#### Mobile
image
image

#### Desktop
image

image

## Testing Steps / QA Criteria
1. On the terminal run ```git checkout main``` to switch to the main branch.
2. Run ```git pull``` the ```git checkout vo-styling-the-list-view```.
3. Run ```npm start``` and view the List view on your browser.

### CODE REVIEW

File: src/views/List.jsx [hunk 1]
```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
...
-		<>
+		<div className="min-h-[100vh]">
 			{isLoading ? (
 				'Loading...'
 			) : data.length ? (
 				<>
 					{listToken && (
-						<div>
+						<div className="flex items-center justify-center gap-2 lg:gap-4 text-[15px] lg:text-[22px] my-3">
 							<p>Your list : {listToken}</p>
```
Comment (klezi10) [inline src/views/List.jsx]: I don't think setting text sizes here is necessary.

I think the text on the list is too large. This is what I see
image

If I remove it from "your list" text, this is how it changes
image

File: src/views/List.jsx [hunk 2]
```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
...
-							<button onClick={copyToken}>{copy}</button>
+							<button
+								className="outline-2 p-2 bg-[#C2410C] rounded-lg"
+								onClick={copyToken}
```
Comment (klezi10) [inline src/views/List.jsx]: I think we should add text-white to the copy token button, it would make the text easier to read.

**Review by klezi10 — CHANGES_REQUESTED** [review]
Nice work @vicowiti , just a few minor changes I think are important.

File: src/views/List.jsx [hunk 3]
```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
...
 				'Loading...'
 			) : data.length ? (
 				<>
 					{listToken && (
-						<div>
+						<div className="flex items-center justify-center gap-2 lg:gap-4 text-[15px] lg:text-[22px] my-3">
 							<p>Your list : {listToken}</p>
-							<button onClick={copyToken}>{copy}</button>
+							<button
+								className="outline-2 p-2 bg-[#C2410C] rounded-lg"
```
Comment (michhub-dev) [inline src/views/List.jsx]: I also feel the copy token text should be white. Looks better that way

File: src/views/List.jsx (hunk 1 above)
Comment (michhub-dev) [reply src/views/List.jsx]: Though no much difference, so it's ok to remove the specified text sizes from the list.

**Review by michhub-dev — APPROVED** [review]

**Review by klezi10 — APPROVED** [review]

**Review by mide358 — APPROVED** [review]

File: src/views/List.jsx [hunk 4]
```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
 		}
 	};
 	return (
-		<>
+		<div className="min-h-[100vh]">
```
Comment (redapy) [inline src/views/List.jsx]: This `min-height` here causes a vertical scroll even if the content is less than the screen size.

**Review by redapy — CHANGES_REQUESTED** [review]
Amazing work!

**Review by redapy — APPROVED** [review]

**Pull request merged**